tests/test_exporters.py

```diff
@@ -427,6 +427,21 @@ def test_nonstring_types_item(self):
         item['time'] = str(item['time'])
         self.assertEqual(exported, item)
 
+    def test_unicode_ascii(self):
+        i1 = TestItem(name=u'Test\u2603')
+        self.assertExportResult(i1, b'{"name": "Test\\u2603"}\n')
+
+    def test_unicode_utf8(self):
+        self.ie = JsonLinesItemExporter(self.output, ensure_ascii = False)
+        i1 = TestItem(name=u'Test\u2603')
+        self.assertExportResult(i1, b'{"name": "Test\xe2\x98\x83"}\n')
```

Comment:
nitpick: I find it a bit easier to read a test against `u'Test\u2603'.encode('utf8')`